db/fns.md

```diff
@@ -2687,6 +2689,26 @@ Otherwise, page_size rows are returned at offset page_offset.
 
 Get an existing worker. The returned table will have one or (if no such worker is defined) zero rows.
 
+### get_worker_pool
```

Comment:
bringing back long deprecated friend, to give info only about the pool itself without extra stats